src/web/server.js

```diff
@@ -125,6 +125,7 @@ async function configureTemplateRendering(instance) {
 
   // make static manifest details available to all templates
   templaterEnvironment.addGlobal('manifest', { ...staticResourceManifest, ...browserManifest })
+  templaterEnvironment.addGlobal('govukRebrand', true) 
```

Comment:
Ah - actually realised we don't need this.
Because Toolbox is a bit different and has it's own layout.njk.

Because you added class - thats all we need.
Can you please remove this.
Thanks